homeassistant/components/intellidrive/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "intellidrive",
+  "name": "Intellidrive",
```

Comment:
A google search for "intellidrive" does not provide results for the sliding door controller, I'd suggest to set the domain to `reisinger_intellidrive` and name the integration "Reisinger Intellidrive".

Reply:
How can I rename the integration without completely recreating it?
The product was released only recently, the website is not yet customized, so you probably did not find anything.

Reply:
Do I only need to customize the manifest file or do I also need to customize the integration folder?

Reply:
Just rename the tests and implementation, stage the changes, and commit 👍
git will pick up the file name change, meaning the history is not lost.

Reply:
Okay but we want to integrate more Home Assistant of our products in Home Assistant in the future though, if we enter the brand name "Reisinger" in the integration search, and then again "Reisinger Intellidrive" is displayed as a result, I would not like it. So should I rename the integration or not?

Reply:
Then we take care of that when you add more Reisinger products by adding a Reisinger brand which has your different products under it 👍 

As an example, this is the Google brand definition:
https://github.com/home-assistant/core/blob/b9c975df694e7f09e954e5bde8cbdefcfa3fbc5a/homeassistant/brands/google.json#L1

This is what it looks like for the user:
![image](https://user-images.githubusercontent.com/14281572/220626531-85e11b49-644d-49ca-a7ce-26da567f2d2d.png)

And then:
![image](https://user-images.githubusercontent.com/14281572/220626597-64a6069d-d748-48f5-858b-71afead384c2.png)

Reply:
wow that worked great, thanks for your help ;)
sorry but I am an absolute noob in Python  🥲